tests/unit_tests/test_lobbyconnection.py

```diff
@@ -120,14 +120,14 @@ def policy_server(loop):
     loop.run_until_complete(runner.cleanup())
 
 
-def test_command_game_host_creates_game(lobbyconnection,
+async def test_command_game_host_creates_game(lobbyconnection,
                                         mock_games,
                                         test_game_info,
                                         players):
     lobbyconnection.player = players.hosting
     players.hosting.in_game = False
     lobbyconnection.protocol = mock.Mock()
-    lobbyconnection.command_game_host(test_game_info)
+    await lobbyconnection.command_game_host(test_game_info)
```

Comment:
I would replace all of these with `lobbyconnection.on_message_received` to make sure the message dispatch actually works. I think putting `timed` on an async function will cause this problem, but the way the tests are written, they will not detect it